openstack/compute/v2/extensions/bootfromvolume/requests.go

```diff
@@ -5,40 +5,65 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
 )
 
-// SourceType represents the type of medium being used to create the volume.
-type SourceType string
+type (
+	// DestinationType represents the type of medium being used as the
+	// destination of the bootable device.
+	DestinationType string
+
+	// SourceType represents the type of medium being used as the source of the
+	// bootable device.
+	SourceType string
+)
 
 const (
-	// Volume SourceType
-	Volume SourceType = "volume"
-	// Snapshot SourceType
-	Snapshot SourceType = "snapshot"
-	// Image SourceType
-	Image SourceType = "image"
 	// Blank SourceType
 	Blank SourceType = "blank"
+
+	// Volume DestinationType
+	DestinationVolume DestinationType = "volume"
+
+	// Image SourceType
+	Image SourceType = "image"
```

Comment:
These should all have either a 'Destination' or 'Source' prefix.
Also, let's group them by type, no alphabetically.